### PR TITLE
Us450849 card brand restriction feature

### PR DESCRIPTION
**Pull request by ochalet-wp**

### CODE REVIEW

File: AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        
+        app.launch()
+        view = NavigationViewPageObject(app).navigateToRestrictedCardFlow()
+    }
+    
+    func testCardNumberTextField_exists() {
+        XCTAssertTrue(view!.panField.exists)
+    }
+    
+    func testCardNumberImageView_exists() {
```
Comment (ochalet-wp) [inline AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift:22]: Rename

File: AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        XCTAssertTrue(view!.cardBrandImage.exists)
+    }
+    
+    func testRestrictedCarBrandsLabel_exists() {
```
Comment (ochalet-wp) [inline AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift:26]: + missing D

File: AccessCheckoutDemo/AccessCheckoutDemoUITests/CardFlowValidationTests.swift [hunk 3]
```diff
@@ -243,4 +243,9 @@ class CardPaymentFlowCardValidationTests: XCTestCase {
         // Invalid state
         XCTAssertFalse(view!.submitButton.isEnabled)
     }
+
+    private func waitFor(timeoutInSeconds: Double) {
```
Comment (ochalet-wp) [inline AccessCheckoutDemo/AccessCheckoutDemoUITests/CardFlowValidationTests.swift:247]: Unused?

File: AccessCheckoutDemo/AccessCheckoutDemoUITests/CvcFlowValidationTests.swift [hunk 4]
```diff
@@ -98,4 +98,9 @@ class CvcFlowCvcValidationTests: XCTestCase {
 
         XCTAssertTrue(view!.submitButton.isEnabled)
     }
+
+    private func waitFor(timeoutInSeconds: Double) {
```
Comment (ochalet-wp) [inline AccessCheckoutDemo/AccessCheckoutDemoUITests/CvcFlowValidationTests.swift:102]: Unused?

Comment (ochalet-wp) [reply AccessCheckoutDemo/AccessCheckoutDemoUITests/CvcFlowValidationTests.swift:102]: Removed

File: AccessCheckoutDemo/AccessCheckoutDemoUITests/CardFlowValidationTests.swift (hunk 3 above)
Comment (ochalet-wp) [reply AccessCheckoutDemo/AccessCheckoutDemoUITests/CardFlowValidationTests.swift:247]: Removed

File: AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift (hunk 1 above)
Comment (ochalet-wp) [reply AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift:22]: Fixed

File: AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift (hunk 2 above)
Comment (ochalet-wp) [reply AccessCheckoutDemo/AccessCheckoutDemoUITests/RestrictedCardFlowUITests.swift:26]: Fixed

**Review by macalac — APPROVED** [review]

**Review by charliewhight — APPROVED** [review]

**Pull request merged**